wpimath/src/main/java/edu/wpi/first/math/trajectory/TrajectoryGenerator.java

```diff
@@ -62,7 +62,7 @@ public static Trajectory generateTrajectory(
       List<Translation2d> interiorWaypoints,
       Spline.ControlVector end,
       TrajectoryConfig config) {
-    final var flip = new Transform2d(new Translation2d(), Rotation2d.fromDegrees(180.0));
+    final var flip = new Transform2d(Translation2d.kZero, Rotation2d.fromDegrees(180.0));
```

Comment:
Should use the Rotation2d constant in this file as well.  Since this same Transform2d constant is used in 3 places in this file, make it a static final variable too?